options/linux/generic/sys-statx.cpp

```diff
@@ -3,9 +3,40 @@
 #include <bits/ensure.h>
 
 #include <mlibc/debug.hpp>
+#include <mlibc/posix-sysdeps.hpp>
 #include <mlibc/linux-sysdeps.hpp>
 
 int statx(int dirfd, const char *pathname, int flags, unsigned int mask, struct statx *statxbuf) {
+	if(!mlibc::sys_statx) {
```

Comment:
Genuine question, does it make more sense for sysdeps to implement this fallback at their own risk? Seems like it could lead to unintended consequences if we add it here

Reply:
According to the man page we should set stx_mask appropriately, then it would be an acceptable catch-all fallback